tsinfer/formats.py

```diff
@@ -2004,33 +2058,92 @@ def num_alleles(self, sites=None):
                 num_alleles[j] -= 1
         return num_alleles[sites]
 
-    def variants(self, sites=None):
+    def variants(self, sites=None, recode_ancestral=None):
         """
         Returns an iterator over the Variant objects. This is equivalent to
-        the TreeSequence.variants iterator.
-        """
+        the TreeSequence.variants iterator. The alleles are guaranteed to be
```

Comment:
Missing a ``If the ``recode_ancestral`` parameter is True, the ..." here

Reply:
Less than clear what happens when there is no ancestral allele defined here. Would help to rephrase the main description of the parameter as a paragraph in the body of the docstring, and reserve the param string for very basic stuff.